app/src/main/java/com/example/trempel/ui/categories/CategoriesAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.trempel.ui.categories
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.trempel.databinding.CategoryItemBinding
+import com.example.trempel.network.CategoryDomainModel
+
+class CategoriesAdapter : RecyclerView.Adapter<CategoriesAdapter.CategoriesViewHolder>() {
```

Comment:
I wonder if it can be marked as `internal`

Reply:
done